sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_receiver_async.py

```diff
@@ -62,8 +62,9 @@ async def _incoming_transfer(self, frame):
         self.received_delivery_id = frame[1] # delivery_id
         # If more is false --> this is the last frame of the message
         if not frame[5]:
-            self.current_link_credit -= 1
             self.delivery_count += 1
+            self.current_link_credit -= 1
```

Comment:
sorry what was the reason we want to keep this line to decrement current_link_credit here again? are there calculations that use this?

Reply:
oh good point nothing should use it for calculations anymore as it is more of a "temporary credit" let me reconfirm that

Reply:
used for sender :)